src/firefly/js/charts/ChartUtil.js

```diff
@@ -545,16 +551,50 @@ export function applyDefaults(chartData={}) {
 
     chartData.layout = merge(defaultLayout, chartData.layout);
 
+    chartData.data && chartData.data.forEach((d, idx) => {
+        d.name = setDefaultName(d, idx);
+
+        const type = get(d, 'type') || get(chartData, ['fireflyData', idx, 'dataType']);
+
+        if (idx === 0 && resetColor) {        // reset the color iterator
+            getNextTraceColor(true);
+            getNextTraceColorscale(true);
+        }
+
+        type && Object.entries(setDefaultColor(d, type)).forEach(([k, v]) => set(d, k, v));
+    });
 }
 
+
 // color-blind friendly colors
-const TRACE_COLORS = [  '#333333', '#ff3333', '#00ccff','#336600',
-                        '#9900cc', '#ff9933', '#009999', '#66ff33', '#cc9999',
-                        '#333333', '#b22424', '#008fb2', '#244700',
+export const TRACE_COLORS = [  '#333333', '#ff3333', '#00ccff', '#336600',
```

Comment:
We might want to use plotly default colors first (without orange), skipping orange and yellow-green, because they are close to our highlight and select colors. What do you think? 


```
'#1f77b4 ',  // muted blue
    '#ff7f0e ',  // safety orange
    '#2ca02c ',  // cooked asparagus green
    '#d62728 ',  // brick red
    '#9467bd ',  // muted purple
    '#8c564b ',  // chestnut brown
    '#e377c2 ',  // raspberry yogurt pink
    '#7f7f7f ',  // middle gray
    '#bcbd22 ',  // curry yellow-green
    '#17becf '   // blue-teal
```

Reply:
the plotly default colors are applied first.